### DISCUSSION TITLE
No results using collection:previous and collection:next ?

### DISCUSSION THREAD

**Discussion by LeoSeyers** [statamic/cms#4703] [Troubleshooting] [answered]
Hello,
Following the [documention](https://statamic.dev/tags/collection-next), I'm trying to use and next and previous tag on collection entity but no results are found. 

I do have multiple entries in my "articles" collection so I'm not sure why it wouldn't work here. 
Publish date are enabled.

Any hint is appreciated 

```vue

{{ collection}} // "articles"
{{ date }}  // November 8th, 2021

{{ collection:next in="articles" as="posts" sort="date:asc" }}

{{ if no_results }}
No more posts to read!
{{ /if }}

{{ posts }}
<div class="post">
    <a href="{{ url }}">{{ title }}</a>
</div>
{{ /posts }}

{{ /collection:next }}
```

**Comment by jesseleite** [comment, marked as answer]
If the current article is Nov 8th, 2021, can you confirm that you have articles that are dated before and after that date? Also what do you get if you throw a corresponding `{{ collection:previous }}` on the same page there?

**Reply by LeoSeyers** [reply]
Indeed all dates were at the same date and so I changed them and it now works properly.
I assumed that it would work anyway but I guess it only do a simple comparaison  which is good enough for my project, in case I can still write my own modifier I presume.

thanks for your help @jesseleite !
